radixdlt-core/radixdlt/src/main/java/org/radix/universe/output/AWSSecretManager.java

```diff
@@ -0,0 +1,176 @@
+        removeBouncyCastleSecurityProvider();
+        Region region = Region.EU_WEST_2;
+        SecretsManagerClient secretsClient = SecretsManagerClient.builder()
+                .region(region)
+                .build();
+
+        updateMySecret(secretsClient, secretName, secretValue);
+        secretsClient.close();
+    }
+
+    private static void updateMySecret(SecretsManagerClient secretsClient, String secretName, String secretValue) {
+        UpdateSecretRequest secretRequest = UpdateSecretRequest.builder()
+                .secretId(secretName)
+                .secretString(secretValue)
+                .build();
+
+        secretsClient.updateSecret(secretRequest);
+    }
+
+    private static String getValue(SecretsManagerClient secretsClient, String secretName) {
+        GetSecretValueRequest valueRequest = GetSecretValueRequest.builder()
+                .secretId(secretName)
+                .build();
+
+        GetSecretValueResponse valueResponse = secretsClient.getSecretValue(valueRequest);
+        return valueResponse.secretString();
+
+    }
+
+    private static String createNewSecret(SecretsManagerClient secretsClient, String secretName, String secretValue, String network) {
+        List<Tag> tagList = buildTags(network, secretName);
+
+        CreateSecretRequest secretRequest = CreateSecretRequest.builder()
+                .name(secretName)
+                .description("Validator keys")
+                .secretString(secretValue)
+                .tags(tagList)
+                .build();
+
+        CreateSecretResponse secretResponse = secretsClient.createSecret(secretRequest);
+        return secretResponse.arn();
+    }
+    public static void createAWSSecret(
+        final Map<String, Object> awsSecret,
+        final String secretName,
+        final AWSSecretsUniverseOutput awsSecretsUniverseOutput
+    ) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        try {
+            String jsonSecret = objectMapper.writeValueAsString(awsSecret);
+            AWSSecretManager.createSecret(secretName, jsonSecret, awsSecretsUniverseOutput.getNetworkName());
+        } catch (JsonProcessingException e) {
+            System.out.println(e);
+        } catch (SecretsManagerException e) {
+            System.err.println(e.awsErrorDetails().errorMessage());
+            System.exit(1);
+        }
+    }
+
+
+    public static void updateAWSSecret(Map<String, Object> awsSecret, String secretName, AWSSecretsUniverseOutput awsSecretsUniverseOutput) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        if (awsSecretsUniverseOutput.getRecreateAwsSecrets()
+            && (!awsSecretsUniverseOutput.getNetworkName().equalsIgnoreCase("betanet")
+                    || !awsSecretsUniverseOutput.getNetworkName().equalsIgnoreCase("mainnet"))) {
+            System.out.format("Secret %s exists. And it's going to be replaced %n", secretName);
```

Comment:
Move following part into dedicated method:
`(!awsSecretsUniverseOutput.getNetworkName().equalsIgnoreCase("betanet")
                    || !awsSecretsUniverseOutput.getNetworkName().equalsIgnoreCase("mainnet"))`
With proper name it will simplify reading the code.